https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/1286
core-util-test.c would fail unless assertions are enabled. Credits to Igor Kovalenko for pointing this out.
The below patch disables the four failing tests.
diff --git a/src/tests/core-util-test.c b/src/tests/core-util-test.c
index 49aa4ab..8cdad19 100644
--- a/src/tests/core-util-test.c
+++ b/src/tests/core-util-test.c
@@ -262,20 +262,26 @@ START_TEST (test_replace) {
 }
 END_TEST
 
+/*
 START_TEST (test_replace_fail_1) {
     pa_replace(NULL, "b", "bab");
 }
 END_TEST
+*/
 
+/*
 START_TEST (test_replace_fail_2) {
     pa_replace("abe", NULL, "bab");
 }
 END_TEST
+*/
 
+/*
 START_TEST (test_replace_fail_3) {
     pa_replace("abcde", "b", NULL);
 }
 END_TEST
+*/
 
 START_TEST (test_escape) {
     char* value;
@@ -294,10 +300,12 @@ START_TEST (test_escape) {
 }
 END_TEST
 
+/*
 START_TEST (test_replace_fail_4) {
     pa_replace("abe", "", "bab");
 }
 END_TEST
+*/
 
 START_TEST (test_unescape) {
     char* value;
@@ -334,10 +342,12 @@ int main(int argc, char *argv[]) {
     tcase_add_test(tc, test_atoi64);
     tcase_add_test(tc, test_atod);
     tcase_add_test(tc, test_replace);
+    /*
     tcase_add_test_raise_signal(tc, test_replace_fail_1, SIGABRT);
     tcase_add_test_raise_signal(tc, test_replace_fail_2, SIGABRT);
     tcase_add_test_raise_signal(tc, test_replace_fail_3, SIGABRT);
     tcase_add_test_raise_signal(tc, test_replace_fail_4, SIGABRT);
+    */
     tcase_add_test(tc, test_escape);
     tcase_add_test(tc, test_unescape);
 
